gluon/tools.py

```diff
@@ -2214,7 +2214,19 @@ def define_tables(self, username=None, signature=None, enable_tokens=False,
             fake_migrate = db._fake_migrate
         settings = self.settings
         settings.enable_tokens = enable_tokens
-        super(Auth, self).define_tables(username, signature, migrate, fake_migrate)
+
+        if not self.signature:
+            self.define_signature()
+        if signature is True:
+            signature_list = [self.signature]
+        elif not signature:
+            signature_list = []
+        elif isinstance(signature, Table):
+            signature_list = [signature]
+        else:
+            signature_list = signature
+
+        super(Auth, self).define_tables(username, signature_list, migrate, fake_migrate)
```

Comment:
why is the code above necessary? authapi.define_tables already has the same exact code